e2e/tests/custom-projects/edit-custom-project.spec.ts

```diff
@@ -29,7 +31,7 @@ test.describe("Custom Projects - Edit", () => {
     await testManager.close();
   });
 
-  test("A user can see two Edit project links with correct URL patterns after creating a custom project", async () => {
+  test("User can see two Edit project links with correct URL patterns after creating a custom project", async () => {
```

Comment:
A little but important suggestion: This is an anti-pattern for e2e test types. The idea for e2e tests are to check the system's behavior by following what a real user would do.
In this case, a user does not care about a URL pattern, it cares about where a link will bring.

If you were to test this behavior, you should test that the link works checking either it goes to a specific route, but even better that loads a expected content.

However, this would make a good unit/component test case. 

I suggest you update this test